src/main/java/bio/terra/workspace/service/create/CreateDAO.java

```diff
@@ -34,6 +34,14 @@ public String create(String workspaceId, JsonNullable<UUID> spendProfile) {
     }
 
     jdbcTemplate.update(sql, paramMap);
-    return workspaceId;
+    return workspaceId.toString();
+  }
+
+  public boolean deleteWorkspace(UUID workspaceId) {
```

Comment:
seems strange to have a "delete" method in a "CreateDAO"

Reply:
+1

I am wondering if we should move all of the DB actions to a centralized DAO that can be shared among the services. Given the current structure, I am writing a GetDAO at the moment which feels a bit strange especially since I could see it being used for other service classes within WS manager. I can imagine there being a lot of duplicated code unless we centralize the common DB code.

Reply:
Actually, that seems like a cleaner solution. It probably makes more sense to have one shared DAO (WorkspaceDAO? though it might also touch other DB tables like resources, data references, etc). I can add that here, but we should coordinate the change for `Get` too

Reply:
Per discussion, we'll move to a shared DAO object for the workspace DB in a separate change.